app/assets/javascripts/components/search/material_info.js.coffee

```diff
@@ -8,7 +8,10 @@ window.SMaterialInfoClass = React.createClass
 
     links = []
     links.push material.links.preview           if material.links.preview
-    links.push material.links.external_edit     if material.links.external_edit
+    if material.lara_activity_or_sequence
+      links.push material.links.external_lara_edit if material.links.external_lara_edit
```

Comment:
The matrix of edit links seems to have the following `link` types:  

* `external_lara_edit`
* `external_edit`
* `external_edit_iframe` 
* `edit`

Some of these links require checking if the material is a `lara_activity_or_sequence` for the purpose of ITSI material display.